lib/ukschedcoop/Config.uk

```diff
@@ -1,4 +1,5 @@
 config LIBUKSCHEDCOOP
 	bool "ukschedcoop: Cooperative Round-Robin scheduler"
 	default y
-	depends on LIBUKSCHED
+	select LIBNOLIBC if !HAVE_LIBC
+	select LIBUKSCHED
```

Comment:
```suggestion
	select LIBUKSCHED

```

Reply:
Should there be a newline here?

Reply:
Yes, an ending newline should be there. I updated my editor settings now. The newline was missing originally but should be there. I fixed it :+1: